[Add-ins](Shared Runtime) Adding that SR uses IE11 (#1902)
OfficeDev/office-js-docs-pr: docs/concepts/browsers-used-by-office-web-add-ins.md

<sup>1</sup> See the [update history page](/officeupdates/update-history-office365-proplus-by-date) and how to [find your Office client version and update channel](https://support.office.com/article/What-version-of-Office-am-I-using-932788b8-a3ce-44bf-bb09-e334518b8b19) for more details.

> >[!NOTE] If your add-in is configured to use a shared runtime, by including the `Runtimes` element in your manifest, your add-in will use Internet Explorer 11 by default. For more information, see [Runtimes](../reference/manifest/runtimes.md).

Reviewer: ```suggestion
> [!NOTE]
> If your add-in is configured to use a shared runtime, by including the `Runtimes` element in your manifest, your add-in will use Internet Explorer 11 by default. For more information, see [Runtimes](../reference/manifest/runtimes.md).
```

Reviewer: Also, I think this note should be placed after the numbered notes.

> [!IMPORTANT]
> Internet Explorer 11 does not support JavaScript versions later than ES5. If any of your add-in's users have platforms that use Internet Explorer 11, then to use the syntax and features of ECMAScript 2015 or later, you will need to either transpile your JavaScript to ES5 or use a polyfill. Also, Internet Explorer 11 does not support some HTML5 features such as media, recording, and location.

>[!NOTE]

> > If your add-in is configured to use a shared runtime, by including the `Runtimes` element in your manifest, your add-in will use Internet Explorer 11 by default. For more information, see [Runtimes](../reference/manifest/runtimes.md).

Reviewer: ```suggestion
> If your add-in includes the `Runtimes` element in your manifest, then it will use Internet Explorer 11 by default. For more information, see [Runtimes](../reference/manifest/runtimes.md).
```
